t/39-dewebsockify.t

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/perl
+
+# Test file for dewebsockify.pm
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+use Test::Most;
+use Test::Warnings;
+use Test::MockObject;
+use Test::Mojo;
+use Mojo::IOLoop;
+use Mojo::IOLoop::Server;
+use FindBin;
+use lib "$FindBin::Bin/../script";
+use_ok('dewebsockify') or die "Cannot load dewebsockify module";
+
+sub mock_logger {
+    my $log = Test::MockObject->new();
+    $log->mock('info', sub { diag "INFO: @_"; });
+    $log->mock('debug', sub { diag "DEBUG: @_"; });
+    $log->mock('error', sub { diag "ERROR: @_"; });
+    $log->mock('trace', sub { diag "TRACE: @_"; });
+    return $log;
+}
```

Comment:
Maybe it is possible to simply use a Mojo log object here (created via `Mojo::Log->new`).